tests/formats/autodetect/zeek.out.zng

```diff
@@ -1,2 +1,2 @@
-#0:record[_path:string,ts:time,uid:string,id:record[orig_h:addr,orig_p:port,resp_h:addr,resp_p:port],version:string,cipher:string,curve:string,server_name:string,resumed:bool,last_alert:string,next_protocol:string,established:bool,cert_chain_fuids:vector[string],client_cert_chain_fuids:vector[string],subject:string,issuer:string,client_subject:string,client_issuer:string,validation_status:string]
+#0:record[_path:string,ts:time,uid:bstring,id:record[orig_h:addr,orig_p:port,resp_h:addr,resp_p:port],version:bstring,cipher:bstring,curve:bstring,server_name:bstring,resumed:bool,last_alert:bstring,next_protocol:bstring,established:bool,cert_chain_fuids:vector[bstring],client_cert_chain_fuids:vector[bstring],subject:bstring,issuer:bstring,client_subject:bstring,client_issuer:bstring,validation_status:bstring]
```

Comment:
why is uid a bstring?

Reply:
Oh, just because short of embedding typing rules that say which zeek string came from the network vs was constructed, we just have to assume everything is a b-string.

Reply:
Correct.  Short of bringing in typings.json from "somewhere else", I don't have any realistic ideas for handling this better.